dipy/align/streamlinear.py

```diff
@@ -877,6 +877,9 @@ def slr_with_qbx(static, moving,
     rm_small_clusters : int, optional
         Remove clusters that have less than `rm_small_clusters` (default 50)
 
+    maxiter : int
```

Comment:
`int, optional` sorry, I forgot this one

Reply:
Done in 409a7e0.